### PR TITLE
[Elixir] broken link

### PR DESCRIPTION
**Pull request by mikedamay**
British english (should) rule(s)

### CODE REVIEW

File: languages/elixir/exercises/concept/access/.docs/after.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
...
+Elixir uses code [_Behaviours_][behaviors] to provide a common generic interfaces while facilitating specific implementations for each module which implements the behavior. When creating a _Behaviour_ in Elixir, the behaviour-defining module contains general logic for a common pattern and which callback functions must be implemented by modules using the behaviour. Modules that implement the _Behaviour_ can then write callback functions which then operate on its own specific needs. Saša Jurić writes in his book, Elixir in Action, that this is an extension of Erlang _Behaviours_ which are commonly seen in modules like [`GenServer`][genserver], [`Supervisor`][supervisor], and [`Application`][application].[1]
```
Comment (neenjaw) [inline languages/elixir/exercises/concept/access/.docs/after.md:1]:
```suggestion
Elixir uses code [_Behaviours_][behaviours] to provide a common generic interfaces while facilitating specific implementations for each module which implements the behavior. When creating a _Behaviour_ in Elixir, the behaviour-defining module contains general logic for a common pattern and which callback functions must be implemented by modules using the behaviour. Modules that implement the _Behaviour_ can then write callback functions which then operate on its own specific needs. Saša Jurić writes in his book, Elixir in Action, that this is an extension of Erlang _Behaviours_ which are commonly seen in modules like [`GenServer`][genserver], [`Supervisor`][supervisor], and [`Application`][application].[1]
```

Comment (neenjaw) [inline languages/elixir/exercises/concept/access/.docs/after.md:1]:
```suggestion
Elixir uses code [_Behaviours_][behaviours] to provide a common generic interfaces while facilitating specific implementations for each module which implements the behavior. When creating a _Behaviour_ in Elixir, the behaviour-defining module contains general logic for a common pattern and which callback functions must be implemented by modules using the behaviour. Modules that implement the _Behaviour_ can then write callback functions which then operate on its own specific needs. Saša Jurić writes in his book, Elixir in Action, that this is an extension of Erlang _Behaviours_ which are commonly seen in modules like [`GenServer`][genserver], [`Supervisor`][supervisor], and [`Application`][application].[1]
```

**Comment by angelikatyborska** [comment]
@mikedamay Will you finish this PR? I have seen that you are active with other PRs, so I am assuming you just forgot :)

**Comment by mikedamay** [comment]
@angelikatyborska sorry about that.

opened #1773 to ensure no merge conflicts

**Pull request closed without merging**